render/gles2/renderer.c

```diff
@@ -227,7 +227,7 @@ static bool gles2_resource_is_wl_drm_buffer(struct wlr_renderer *wlr_renderer,
 static void gles2_wl_drm_buffer_get_size(struct wlr_renderer *wlr_renderer,
 		struct wl_resource *buffer, int *width, int *height) {
 	struct wlr_gles2_renderer *renderer =
-		gles2_get_renderer_in_context(wlr_renderer);
+		gles2_get_renderer(wlr_renderer);
```

Comment:
Not sure if we should switch context here. @ascent12, thoughts?

Reply:
I think it should be as it is now. I've also retested reverting this change - after some working in sway and opening some XWayland apps the assert is raising.